activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Generates a method attribute_salt that returns the salt used to hash the password
+    generated by `has_secure_password`, it can be used for tracking changes
+    and as the source for `generates_token_for`.
```

Comment:
What do you think about rewording this a bit to clarify the purpose?

```suggestion
*   `has_secure_password` now generates an `#{attribute}_salt` method that 
    returns the salt used to compute the password digest.

    The salt will change whenever the password is changed, so it can be used to 
    create single-use password reset tokens with `generates_token_for`:
```